pyproject.toml

```
@@ -243,6 +243,7 @@ doctest_subpackage_requires = [
     "astropy/table/mixins/dask.py = dask",
     "docs/* = numpy>=2",   # not NUMPY_LT_2_0 (PR 15065)
     "docs/units/type_hints.rst = python<=3.13",  # PYTHON_LT_3_14 (PR 18140)
+    "docs/units/structured_units.rst = erfa<=2.1.0.7",  # PYERFA_LT_2.1.0.8
```

Has to be installable from PyPI right? https://pypi.org/project/pyerfa/

```suggestion
    "docs/units/structured_units.rst = pyerfa<=2.1.0.7",  # PYERFA_LT_2.1.0.8
```